### ISSUE TITLE
[BUG] sof-kernel-log-check: filter out IGB issues

### ISSUE THREAD

**Issue by plbossart** [thesofproject/sof-test#617]
One more issue on UpExtreme, solved with following string:

````
+ignore_str="$ignore_str"'|igb 0000:0.:00\.0: exceed max 2 second'
````

Maybe we need to ignore all IGB problems really.

**Comment by marc-hb** [comment]
`igb` = Intel Gigabit Ethernet.

I searched this message in the source code and on the Internet and it could be a cable issue, can you try a different cable?

Ignoring bad cables and similar network issues sounds like a bad idea.

**Comment by plbossart** [comment]
It's not a cable issue, I use the same with other interfaces (RTL or USB dongles) and they work fine.

**Comment by marc-hb** [comment]
Gigabit Ethernet uses all 4 pairs both ways. 100Mb/s uses only 2 pairs one-way each.

**Comment by plbossart** [comment]
I have only Gigabit stuff and 935Mbs/s all the way to my devices over Google Fiber. Cables are not the problem

**Comment by plbossart** [comment]
PING:
6.0 ms
DOWNLOAD
936 Mbps
UPLOAD
816 Mbps

**Comment by marc-hb** [comment]
Looking at `drivers/net/ethernet/intel/igb/igb_main.c` this error message is printed when and only when a watchdog found `SR_1000T_REMOTE_RX_STATUS` a.k.a `Remote receiver OK` to be repeatedly wrong 20 times over 2 seconds. Really not good.

How often do you get this message and what other `igb` messages are around it? It could have been just a temporary glitch because of some bad contact or bent cable or temporary interference.

I think we really need this custom sof-test config file where anyone can add any particular error they have. Of course CI would never use such custom additions but only the default and common set of ignored errors. Using such a file would print a warning at the start of the test. It's one thing to ignore network or other glitches at home when you can just stand up, give the system a kick, re-run the tests without even looking at what went wrong and forget about it. It's a totally different story to ignore network glitches in a distant lab, especially during COVID. @fredoh9 and others have already spent _many days_ in the lab troubleshooting various storage and network issues and hiding warning signs is really not a good idea. Ignoring storage and network errors is not like ignoring wifi or keyboard errors. 

(even funnier, ignoring audio errors: https://github.com/thesofproject/sof/pull/2686)

**Comment by plbossart** [comment]
I get this with a different board and different cables

````
===========================>>
[  936.832321] kernel: igb 0000:01:00.0 enp1s0: Reset adapter
[  938.816256] kernel: igb 0000:01:00.0 enp1s0: Reset adapter
[  940.800272] kernel: igb 0000:01:00.0 enp1s0: Reset adapter
<<===========================
````

**Comment by marc-hb** [comment]
@plbossart closed by #631?
